### PR TITLE
SPCTR-2390 Carousel - TwentyTwentyone theme hover style override for arrow and dot.

### PR DESCRIPTION
**Pull request by ravinishadcoder**
### Description

Post Carousel - TwentyTwentyone theme hover style override for arrow and dot.
### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — CHANGES_REQUESTED** [review]
Please pull in next-release

**Review by AbhijitNage123 — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Comment by aapatre** [comment]
Please add 'Post Carousel' to heading in place of 'Carousel'

File: src/blocks/post/style.scss [hunk 1]
```diff
@@ -333,6 +333,7 @@
 		border: none;
 		display: inline-block;
 		background: none;
+		color: #fff;
```
Comment (supriyajjadhav) [inline src/blocks/post/style.scss:336]: If we add static CSS to apply color then the theme's default color will not work.

File: src/blocks/post/post-button/edit.js [hunk 2]
```diff
@@ -1,19 +1,11 @@
...
 export const PostButton = ( props ) => {
 	// Add and remove the CSS on the drop and remove of the component.
-	useLayoutEffect( () => {
-		styles.use();
-		return () => {
-			styles.unuse();
-		};
-	}, [] );
-
+
```
Comment (AbhijitNage123) [inline src/blocks/post/post-button/edit.js:8]: Is there any reason to delete this?

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

File: src/blocks/post/post-button/edit.js [hunk 3]
```diff
@@ -1,19 +1,11 @@
 import { __ } from '@wordpress/i18n';
 import classnames from 'classnames';
-import { useLayoutEffect } from '@wordpress/element';
-import styles from './editor.lazy.scss';
 import { RichText } from '@wordpress/block-editor';
 
 
 export const PostButton = ( props ) => {
 	// Add and remove the CSS on the drop and remove of the component.
-	useLayoutEffect( () => {
```
Comment (AbhijitNage123) [inline src/blocks/post/post-button/edit.js:10]: Is there any reason to delete this?

Comment (ravinishadcoder) [reply src/blocks/post/post-button/edit.js:10]: I had tested at my end. CSS was not loaded anywhere on the page of these files. So I discussed with Supriya and removed the editor.lazy.scss file for post-image,post-meta and post-title

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by supriyajjadhav — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**